### ISSUE TITLE
No suggestions in VS Code

### ISSUE THREAD

**Issue by foxpluto** [snyk/vscode-extension#236]
I have just installed the extension in VSCode but I can't see the suggestions:

```
[Error] Failed to show Snyk Code suggestion view. {"stack":"Error: Webview is disposed\n\tat h.assertNotDisposed (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:83:290246)\n\tat h.set title [as title] (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:83:289206)\n\tat CodeSuggestionWebviewProvider.showPanel (/Users/fox/.vscode/extensions/snyk-security.snyk-vulnerability-scanner-1.2.18/out/snyk/snykCode/views/suggestion/codeSuggestionWebviewProvider.js:93:34)\n\tat runMicrotasks (<anonymous>)\n\tat processTicksAndRejections (node:internal/process/task_queues:96:5)","message":"Webview is disposed"}
```

VSCode Version: 1.68.1
Snyk: 1.2.18

**Comment by TheTFo** [comment]
I can concur on this, and see it as well with Code Security and Code Quality items.  I can reproduce fairly consistently by clicking and viewing a suggestion in the Webview window and closing it, and repeating that several time. After awhile, I get the error from OP.

Using the same versions as OP:

VSCode Version: 1.68.1
Snyk: 1.2.18

**Comment by michelkaporin** [comment]
Sorry you're experiencing this. It's on our deck to fix soon.

**Comment by TheTFo** [comment]
No worries! I appreciate you looking into it

**Comment by michelkaporin** [comment]
Should be solved after #248 now.
